indexer/validate_and_format_contacts.cpp

```diff
@@ -230,8 +235,13 @@ bool ValidateFacebookPage(string const & page)
   if (page.empty())
     return true;
 
-  // Rules are defined here: https://www.facebook.com/help/105399436216001
-  if (regex_match(page, s_fbRegex))
+  // Check if 'page' contains valid Facebook username or page name.
+  // * length >= 5
+  // * no forbidden symbols in the string
+  // * optional '@' at the start
+  if (page.front() == '@')
+    return page.length() >= 6 && page.substr(1).find_first_of(kForbiddenFBSymbols) == string::npos;
```

Comment:
Not making additional substr copy: page.find_first_of(kForbiddenFBSymbols, 1)

Reply:
Removed `page.substr(1)`